diff --git a/web/profiles/elvaco/modules/features/features.module b/web/profiles/elvaco/modules/features/features.module
index 2756bd5..4cb8b13 100644
--- a/web/profiles/elvaco/modules/features/features.module
+++ b/web/profiles/elvaco/modules/features/features.module
@@ -213,7 +213,10 @@ function features_theme() {
  * Implements hook_flush_caches().
  */
 function features_flush_caches() {
-  features_rebuild();
+  global $install_state;
+  if (!is_array($install_state)) {
+    features_rebuild();
+  }
   features_get_modules(NULL, TRUE);
   return array();
 }
